.github/workflows/ci_tests_dev.yaml

```
@@ -185,5 +185,5 @@ jobs:
         uses: actions/upload-artifact@v4.6.0
         if: ${{ failure() }}
         with:
-          name: artifact-GMT-${{ matrix.gmt_git_ref }}-${{ runner.os }}
+          name: artifact-GMT-${{ matrix.gmt_git_ref }}-${{ matrix.os }}
```

Update the artifact name so it's consistent with the one in `ci_tests.yaml`?
```suggestion
          name: artifact-${{ matrix.os }}-GMT-${{ matrix.gmt_git_ref }}
```